plugin/pkg/scheduler/core/generic_scheduler.go

```diff
@@ -44,7 +46,10 @@ type FitError struct {
 
 var ErrNoNodesAvailable = fmt.Errorf("no nodes available to schedule pods")
 
-const NoNodeAvailableMsg = "No nodes are available that match all of the following predicates:"
+const (
+	NoNodeAvailableMsg = "No nodes are available that match all of the following predicates:"
```

Comment:
Would you mind remove the semicolon? as there is already one here:
`reasonMsg := fmt.Sprintf(NoNodeAvailableMsg+": %v.", strings.Join(sortReasonsHistogram(), ", "))`